deployment/helm/resource-management-policies/topology-aware/values.yaml

```diff
@@ -9,8 +9,11 @@ image:
   pullPolicy: Always
 
 config:
-  ReservedResources:
-    cpu: 750m
+  policy:
+    ReservedResources:
+      cpu: 750m
+  logger:
+    Debug: nri-resource-policy,resource-manager,cache,policy,topology-hints
```

Comment:
What is this for ? I mean we don't have it for balloons by default, what''s the motivation to have logger enabled by default?

Reply:
ah, that is a mistake. I'll fix.

Reply:
fixed